readme.md

```diff
@@ -52,111 +53,119 @@ Once you search you should see a list similar to the one below (versions may be
 
-### Code Helpers
-* [AdvancedCollectionView](http://docs.uwpcommunitytoolkit.com/en/master/helpers/AdvancedCollectionView/)
-* [AppPinManager](http://docs.uwpcommunitytoolkit.com/en/master/helpers/AppPinManager/)
-* [BackgroundTaskHelper](http://docs.uwpcommunitytoolkit.com/en/master/helpers/BackgroundTaskHelper/)
-* [BindableValueHolder](http://docs.uwpcommunitytoolkit.com/en/master/helpers/BindableValueHolder/)
-* [BluetoothLEHelper](http://docs.uwpcommunitytoolkit.com/en/master/helpers/BluetoothLEHelper/)
-* [Colors](http://docs.uwpcommunitytoolkit.com/en/master/helpers/Colors/)
-* [Converters](http://docs.uwpcommunitytoolkit.com/en/master/helpers/Converters/)
-* [DeepLinkParsers](http://docs.uwpcommunitytoolkit.com/en/master/helpers/DeepLinkParsers/)
-* [DispatcherHelper](http://docs.uwpcommunitytoolkit.com/en/master/helpers/DispatcherHelper/)
-* [HttpHelper](http://docs.uwpcommunitytoolkit.com/en/master/helpers/HttpHelper/)
-* [ImageCache](http://docs.uwpcommunitytoolkit.com/en/master/helpers/ImageCache/)
-* [Incremental Loading Collection](http://docs.uwpcommunitytoolkit.com/en/master/helpers/IncrementalLoadingCollection/)
-* [NetworkHelper](http://docs.uwpcommunitytoolkit.com/en/master/helpers/NetworkHelper/)
-* [Object Storage](http://docs.uwpcommunitytoolkit.com/en/master/helpers/ObjectStorage/)
-* [PrintHelper](http://docs.uwpcommunitytoolkit.com/en/master/helpers/PrintHelper/)
-* [StorageFiles](http://docs.uwpcommunitytoolkit.com/en/master/helpers/StorageFiles/)
-* [Streams](http://docs.uwpcommunitytoolkit.com/en/master/helpers/Streams/)
-* [SystemInformation](http://docs.uwpcommunitytoolkit.com/en/master/helpers/SystemInformation/)
-* [WeakEventListener](http://docs.uwpcommunitytoolkit.com/en/master/helpers/WeakEventListener/)
+* [Composition Visual Attached Properties](https://docs.microsoft.com/en-us/windows/uwpcommunitytoolkit/extensions/VisualEx)
+* [Hyperlink](https://docs.microsoft.com/en-us/windows/uwpcommunitytoolkit/extensions/Hyperlink)
+* [ListViewBase](https://docs.microsoft.com/en-us/windows/uwpcommunitytoolkit/extensions/ListViewBase)
+* [Logical Tree](https://docs.microsoft.com/en-us/windows/uwpcommunitytoolkit/extensions/LogicalTree)
+* [Mouse Cursor](https://docs.microsoft.com/en-us/windows/uwpcommunitytoolkit/extensions/MouseCursor)
+* [SurfaceDialTextbox](https://docs.microsoft.com/en-us/windows/uwpcommunitytoolkit/extensions/SurfaceDialTextboxHelper)
+* [TextBoxMask](https://docs.microsoft.com/en-us/windows/uwpcommunitytoolkit/extensions/TextBoxMask)
+* [TextBoxRegex](https://docs.microsoft.com/en-us/windows/uwpcommunitytoolkit/extensions/TextBoxRegex)
+* [ViewExtensions](https://docs.microsoft.com/en-us/windows/uwpcommunitytoolkit/extensions/ViewExtensions)
+* [VisualTree](https://docs.microsoft.com/en-us/windows/uwpcommunitytoolkit/extensions/VisualTree)
+* [WebView](https://docs.microsoft.com/en-us/windows/uwpcommunitytoolkit/extensions/WebView)
 
-### Services
+### Code Helpers
```

Comment:
I think Helpers would be a better name than Code Helpers because it will match the NuGet package name and in other places
![chrome_2017-12-30_09-19-33](https://user-images.githubusercontent.com/24755596/34451127-9c5e887c-ed42-11e7-84a7-99e3e38d8465.png)

If not, we should move Code Helpers to second place to match alphabetical order.